frontend/src/components/Login/RedirectIfLoggedIn.js

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Cookies from 'js-cookie';
+
+function RedirectIfLoggedIn({ }) {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const checkLoggedIn = async () => {
```

Comment:
Don't define function inside useEffect, define this function about RedirectIfLoggedIn component